### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Svitlana007**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 - [Marketing automation platform wich have ecommerce integration](#marketing-automation-platform-wich-have-ecommerce-integration)
 - [Automation Email Marketing platform](#automation-email-marketing-platform)
 - [Tools for effective works with e-mail](#tools-for-effective-works-with-e-mail)
+- [Strategy](#strategy)
+- [Similar companies](#similar-companies)
+- [Building a list](#building-a-list)
+- [Data Driven Email Mark](#data-driven-email-mark)
```
Comment (atherdon) [inline README.md:13]: can you change it to "Data Driven Email Marketing"

File: README.md [hunk 2]
```diff
@@ -7,6 +7,14 @@
 - [Marketing automation platform wich have ecommerce integration](#marketing-automation-platform-wich-have-ecommerce-integration)
 - [Automation Email Marketing platform](#automation-email-marketing-platform)
 - [Tools for effective works with e-mail](#tools-for-effective-works-with-e-mail)
+- [Strategy](#strategy)
+- [Similar companies](#similar-companies)
+- [Building a list](#building-a-list)
+- [Data Driven Email Mark](#data-driven-email-mark)
+- [Lifecycle of](#lifecycle-of)
```
Comment (atherdon) [inline README.md:14]: can you update it, please?

File: README.md [hunk 3]
```diff
@@ -125,6 +133,99 @@
...
 
+## Strategy
+
+[How Often Should You Send Your Subscription Newsletter?](https://bloggingguide.com/how-often-should-you-send-your-subscription-newsletter/)
+
+## Similar companies
+
+[Feedotter](https://www.feedotter.com/)
+
+## Building a list
```
Comment (atherdon) [inline README.md:144]: "Building an email list"

**Review by atherdon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Svitlana007) [reply README.md:13]: Done

File: README.md (hunk 3 above)
Comment (Svitlana007) [reply README.md:144]: Done

File: README.md (hunk 2 above)
Comment (Svitlana007) [reply README.md:14]: Done

**Pull request merged**